agendas/2018-02-01.md

```diff
@@ -44,6 +44,8 @@ Mike Marcacci        | Boltline      | San Diego, CA
 1. Determine volunteers for note taking (5m)
 1. Discuss upcoming meeting schedule (5m)
 1. Present [graphql-import](https://github.com/graphcool/graphql-import) (5m)
+1. Discuss exposing schema metadata (30m)
+1. Discuss errors (30m)
```

Comment:
I know this has been discussed previously, feel free to remove it if you feel like this won't add much to the conversation 👍